tests/Unit/Filament/Resources/Locations/LocationResourceTest.php

```
@@ -0,0 +1,67 @@
+    public function should_return_correct_model_class(): void
+    {
+        $resource = new LocationResource;
+        $modelProperty = new \ReflectionProperty(LocationResource::class, 'model');
+
+        $this->assertSame(Location::class, $modelProperty->getDefaultValue());
+    }
+
+    #[\PHPUnit\Framework\Attributes\Test]
+    public function should_return_correct_navigation_icon(): void
+    {
+        $resource = new LocationResource;
```

PHPMD (2.15.0)

Avoid unused local variables such as '$resource'. (undefined)

(UnusedLocalVariable)